mawk.mk

```diff
@@ -0,0 +1,36 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+SUBPROJECTS   += mawk
+MAWK_VERSION := 1.3.4
+MAWK_COMMIT := 20200120
+DEB_MAWK_V   ?= $(MAWK_VERSION)
+mawk-setup: setup
+	wget -q -nc -P$(BUILD_SOURCE) http://deb.debian.org/debian/pool/main/m/mawk/mawk_$(MAWK_VERSION).$(MAWK_COMMIT).orig.tar.gz
```

Comment:
Add a space after line 8. Add a space after the argument, -P; improves readability.